tests/test_default_router.py

```diff
@@ -447,6 +447,8 @@ def test_literal(router):
     )
 ])
 def test_converters(router, path, expected_params):
+    with open('finder.py', 'w') as f:
```

Comment:
Is this output file later analyzed in any way, or is it a debugging leftover?

Reply:
that's also a leftover..